--- ../src/main/java/io/github/offbeat_stuff/zombie_apocalypse/ArmorTrimHander.java.orig	2023-06-13 15:38:28.029999711 +0530
+++ ../src/main/java/io/github/offbeat_stuff/zombie_apocalypse/ArmorTrimHander.java	2023-06-13 15:58:12.639999530 +0530
@@ -10,7 +10,6 @@
 import net.minecraft.registry.Registry;
 import net.minecraft.registry.RegistryKeys;
 import net.minecraft.registry.entry.RegistryEntry;
-import net.minecraft.resource.featuretoggle.FeatureFlags;
 import net.minecraft.server.world.ServerWorld;
 import net.minecraft.util.Identifier;
 import net.minecraft.util.math.MathHelper;
@@ -43,10 +42,6 @@
   }
 
   public static void applyRandomArmorTrim(ServerWorld world, ItemStack istack) {
-    if (!(world.getEnabledFeatures().contains(FeatureFlags.UPDATE_1_20))) {
-      return;
-    }
-
     if (!(istack.getItem() instanceof ArmorItem)) {
       return;
     }
