diff --git a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ArmorHandler.java b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ArmorHandler.java
index 411d62d..fa1e35c 100644
--- a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ArmorHandler.java
+++ b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ArmorHandler.java
@@ -2,7 +2,6 @@ package io.github.offbeat_stuff.zombie_apocalypse.config;
 
 import static io.github.offbeat_stuff.zombie_apocalypse.ProbabilityHandler.tryChance;
 
-import io.github.offbeat_stuff.zombie_apocalypse.ArmorTrimHandler;
 import io.github.offbeat_stuff.zombie_apocalypse.ProbabilityHandler.ChanceList;
 import io.github.offbeat_stuff.zombie_apocalypse.ProbabilityHandler.WeightList;
 import java.util.List;
@@ -46,7 +45,6 @@ public class ArmorHandler {
   private static ItemStack randomArmor(ServerWorld world, List<Item> items,
                                        List<Float> chances) {
     var r = Common.randomEnchanctedItemStack(items, chances);
-    ArmorTrimHandler.applyRandomArmorTrim(world, r);
     return r;
   }
 
