diff --git a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/Config.java b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/Config.java
index e42c82a..eb79f8e 100644
--- a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/Config.java
+++ b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/Config.java
@@ -3,8 +3,6 @@ package io.github.offbeat_stuff.zombie_apocalypse.config;
 import static io.github.offbeat_stuff.zombie_apocalypse.ProbabilityHandler.*;
 import static io.github.offbeat_stuff.zombie_apocalypse.config.Common.*;
 
-import io.github.offbeat_stuff.zombie_apocalypse.ArmorTrimHandler;
-import io.github.offbeat_stuff.zombie_apocalypse.ArmorTrimHandler.RawTrimHandler;
 import io.github.offbeat_stuff.zombie_apocalypse.PotionEffectHandler.RawStatusEffectHandler;
 import io.github.offbeat_stuff.zombie_apocalypse.config.SpawnHandler.SpawnConfig;
 import java.util.List;
@@ -37,13 +35,6 @@ public class Config {
           new WeightList(List.of(100, 20, 20, 75, 1), 100),
           new WeightList(List.of(1, 10, 100, 50, 50, 100), 100), 0.1f);
 
-  public RawTrimHandler ArmorTrims = new RawTrimHandler(
-      ArmorTrimHandler.vanillaPatterns, ArmorTrimHandler.vanillaMaterials,
-      new WeightList(List.of(), 110),
-      new WeightList(List.of(100, 100, 1, 50, 100, 100, 100, 10, 100, 100),
-                     100),
-      true, 0.1f);
-
   // Enchantment levels for armor and weapons
   public Range enchantmentLevelRange = new Range(5, 40);
 
