diff --git a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ConfigHandler.java b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ConfigHandler.java
index ff15d9e..faf94eb 100644
--- a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ConfigHandler.java
+++ b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/ConfigHandler.java
@@ -3,7 +3,6 @@ package io.github.offbeat_stuff.zombie_apocalypse.config;
 import static io.github.offbeat_stuff.zombie_apocalypse.config.Common.*;
 import static net.minecraft.util.math.MathHelper.clamp;
 
-import io.github.offbeat_stuff.zombie_apocalypse.ArmorTrimHandler;
 import io.github.offbeat_stuff.zombie_apocalypse.PotionEffectHandler;
 import java.util.List;
 import java.util.function.Predicate;
@@ -33,7 +32,6 @@ public class ConfigHandler {
     ArmorHandler.handleRawArmorHandler(config.Armor);
     WeaponHandler.handleRawWeaponHandler(config.Weapon);
     PotionEffectHandler.handleRaw(config.statusEffects);
-    ArmorTrimHandler.handleRawTrimHandler(config.ArmorTrims);
 
     SpawnHandler.handle(config.spawn);
 
