diff --git a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java
index ab79615..1e172d9 100644
--- a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java
+++ b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java
@@ -140,7 +140,8 @@ public class SpawnHandler {
                               minPlayerDistance)) {
       return false;
     }
-    var zombie = EntityType.ZOMBIE.spawn(world, pos, SpawnReason.NATURAL);
+    var zombie = EntityType.ZOMBIE.spawn(world, null, null, pos,
+                                         SpawnReason.NATURAL, false, false);
     PotionEffectHandler.applyRandomPotionEffects(zombie);
     if (!world.isSpaceEmpty(zombie) ||
         !world.doesNotIntersectEntities(zombie)) {
