diff --git a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java
index 38b2cf6..d948561 100644
--- a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java
+++ b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/config/SpawnHandler.java
@@ -15,9 +15,9 @@ import net.minecraft.entity.Entity.RemovalReason;
 import net.minecraft.entity.EntityType;
 import net.minecraft.entity.SpawnGroup;
 import net.minecraft.entity.SpawnReason;
-import net.minecraft.registry.tag.BlockTags;
 import net.minecraft.server.network.ServerPlayerEntity;
 import net.minecraft.server.world.ServerWorld;
+import net.minecraft.tag.BlockTags;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.util.math.Direction;
 import net.minecraft.util.math.Direction.Axis;
@@ -140,7 +140,8 @@ public class SpawnHandler {
                               minPlayerDistance)) {
       return false;
     }
-    var zombie = EntityType.ZOMBIE.spawn(world, pos, SpawnReason.NATURAL);
+    var zombie = EntityType.ZOMBIE.spawn(world, null, null, pos,
+                                         SpawnReason.NATURAL, false, false);
     PotionEffectHandler.applyRandomPotionEffects(zombie);
     if (!world.isSpaceEmpty(zombie) ||
         !world.doesNotIntersectEntities(zombie)) {
