diff --git a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/ArmorTrimHandler.java b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/ArmorTrimHandler.java
index 74f3876..0b7807f 100644
--- a/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/ArmorTrimHandler.java
+++ b/src/main/java/io/github/offbeat_stuff/zombie_apocalypse/ArmorTrimHandler.java
@@ -10,7 +10,6 @@ import net.minecraft.item.trim.ArmorTrim;
 import net.minecraft.registry.Registry;
 import net.minecraft.registry.RegistryKeys;
 import net.minecraft.registry.entry.RegistryEntry;
-import net.minecraft.resource.featuretoggle.FeatureFlags;
 import net.minecraft.server.world.ServerWorld;
 import net.minecraft.util.Identifier;
 import net.minecraft.util.math.MathHelper;
@@ -43,10 +42,6 @@ public class ArmorTrimHandler {
   }
 
   public static void applyRandomArmorTrim(ServerWorld world, ItemStack istack) {
-    if (!(world.getEnabledFeatures().contains(FeatureFlags.UPDATE_1_20))) {
-      return;
-    }
-
     if (!(istack.getItem() instanceof ArmorItem)) {
       return;
     }
